dashdotdb/controllers/token.py

```diff
@@ -21,7 +21,7 @@
 }
 
 
-def auth_token(token, required_scopes):
```

Comment:
This is a function signature for oauth and I'm not sure if we can remove the `requires_scopes`.  It's empty unless oauth is used unfortunately.  Can we exclude this from checks?